scripts/bootstrap-ansible.sh

```diff
@@ -75,8 +75,13 @@ pushd ${OA_DIR}
   # path for Ansible to search.
   sed -i "s|/etc/ansible/roles:|/etc/ansible/roles:${RPCD_DIR}/playbooks/roles:|" /usr/local/bin/openstack-ansible.rc
 
-  # Now use GROUP_VARS of OSA and RPC
-  sed -i "s|GROUP_VARS_PATH=.*|GROUP_VARS_PATH=\"\${GROUP_VARS_PATH:-${OA_DIR}/playbooks/inventory/group_vars/:${BASE_DIR}/group_vars/:/etc/openstack_deploy/group_vars/}\"|" /usr/local/bin/openstack-ansible.rc
-  sed -i "s|HOST_VARS_PATH=.*|HOST_VARS_PATH=\"\${HOST_VARS_PATH:-${OA_DIR}/playbooks/inventory/host_vars/:${BASE_DIR}/host_vars/:/etc/openstack_deploy/host_vars/}\"|" /usr/local/bin/openstack-ansible.rc
+  # NOTE(cloudnull): Sync {group,host}_vars from rpc-o into the proper OSA
```

Comment:
Doing this will mean that our support group won't be able to change these files or add any other overrides to these group vars. If they're OK with that, then I guess this is OK. I would prefer it if the files were deliberately set to be read-only just to make it a little clearer.

Reply:
++ I'll make them 0400.

Reply:
done.